android/src/main/kotlin/com/airship/flutter/AirshipPlugin.kt

```diff
@@ -362,7 +362,8 @@ class AirshipPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
                     try {
```

Comment:
we should not have a try here, its already handled by the resolve method extension